src/components/VNotificationBanner.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div
+    class="wrapper px-4 md:px-7 py-2 md:flex md:items-center md:justify-between"
+    dir="ltr"
```

Comment:
Does this `dir` need to be always "ltr"? Doesn't this change for the Arabic language for example?

Reply:
I don't remember why this was added 🙈